examples/chef/README.md

```diff
@@ -150,7 +150,7 @@ chef_$PLATFORM:
               scripts/checkout_submodules.py --allow-changing-global-git-config --shallow --platform $PLATFORM
         - name: Bootstrap
           timeout-minutes: 25
-          run: scripts/build/gn_bootstrap.sh
+          run: bash -c "source scripts/bootstrap.sh"
```

Comment:
Why "source" in the subshell, if it's going to exit anyway?  That is, why not just `bash scripts/bootstrap.sh`?